transformations/underscore_trick/transformation.py

```diff
@@ -0,0 +1,54 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def add_underscore(text, new_symbol="_", prob=0.05, seed=42, max_outputs=1):
+    random.seed(seed)
+
+    perturbed_texts = []
+    for _ in itertools.repeat(None, max_outputs):
+        perturbed_text = "".join(
+            [
+                letter
+                if letter != " " or random.random() > prob
+                else new_symbol
+                for letter in text
+            ]
+        )
+        perturbed_texts.append(perturbed_text)
+    return perturbed_texts
+
+
+class UnderscoreTrick(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["All"]
+    keywords = [
+        "morphological",
```

Comment:
Keywords: I'm not sure if this can be considered change can be considered morphological. noise seems appropriate.